measure_extinction/utils/plot_bandpasses.py

```diff
@@ -0,0 +1,23 @@
+from astropy.table import Table
+from matplotlib import pyplot as plt
+import pkg_resources
+from synphot import SpectralElement
+
+# path for band response curves
+band_path = pkg_resources.resource_filename(
+    "measure_extinction", "data/Band_RespCurves/")
+
+# Define the different bandpasses
+bands = ["JohnU", "JohnB", "JohnV", "JohnR", "JohnI", "JohnJ", "2MASSJ", "JohnH", "2MASSH", "JohnK", "JohnKs", "2MASSKs", "AAOL", "IRAC1", "AAOLprime", "IRAC2", "AAOM", "IRAC3", "IRAC4"]
+
+# Plot all response curves
+for band in bands:
+    bp = SpectralElement.from_file("%s%s.dat" % (band_path, band))
+    if "IRAC" in band or "MIPS" in band:
+        wavelengths = bp.waveset * 10**4
+    else:
+        wavelengths = bp.waveset
+    plt.plot(wavelengths, bp(bp.waveset), label=band)
+
+plt.legend()
+plt.show()
```

Comment:
All this needs to move into a function and then a standard `if __name__ == '__main__':` added at the bottom to run it as a script.  The current setup of this file is causing the online tests to fail.

Reply:
Done.